### ISSUE TITLE
Missing Keyboard Access

### ISSUE THREAD

**Issue by normanbrobinson** [creativetimofficial/material-dashboard#67]
I am expecting keyboard access to the demo dashboards.

Currently, only the "get more space" and task dashboard receive focus. Labels are missing from multiple elements. This makes these dashboards difficult or impossible to use with assistive technologies such as screen readers or screen magnifiers that rely upon focus.

Simply using keyboard only and verifying functionality without touch or mouse, including visual focus illustrates the issue.

Chome Lighthouse addon audit for accessibility begins to show some of these issues.

**Comment by groovemen** [comment]
Hello @normanbrobinson, 

Thank you for your interest in working with our products, I just run some tests with the Lighthouse Addon and I will present the report based on that test to the team and we'll find a solution to have perfect accessibilities for our dashboards.

We appreciate your attention to details and also that you have informed us about this issue.

All the best, 
Stefan
